src/ucp/proto/proto_am.inl

```diff
@@ -521,14 +521,18 @@ static UCS_F_ALWAYS_INLINE ucp_request_t*
 ucp_proto_ssend_ack_request_alloc(ucp_worker_h worker, ucs_ptr_map_key_t ep_id)
 {
     ucp_request_t *req;
+    ucp_ep_h ep;
 
+    ep  = UCP_WORKER_GET_EP_BY_ID(worker, ep_id, 0, return NULL,
```

Comment:
why no check for closed ep?

Reply:
it fails the tests, since we closed the EP and it is in-progress, but can send an ACK for send
if not, it will fail in progressing the send operation

Reply:
why do we expect to get an ack in that test?